docs/stories/getting-started/usage-examples/top-app-bar-as-container.stories.js

```diff
@@ -0,0 +1,104 @@
+import '@vonage/vwc-side-drawer';
+import '@vonage/vwc-top-app-bar';
+import '@vonage/vwc-top-app-bar-fixed';
+import '@vonage/vwc-icon';
+import '@vonage/vwc-layout';
+import '@vonage/vwc-list';
+import '@vonage/vwc-list/vwc-list-item';
+import '@vonage/vwc-list/vwc-list-expansion-panel';
+import { html } from 'lit-element';
+import { spread } from '@open-wc/lit-helpers';
+
+export default {
+	title: 'Getting Started/Usage Examples/Top App Bar',
+};
+
+const style = html`
+	<style>
+		.sb-show-main.sb-main-padded {
+			padding: 0;
+		}
+		p {
+			color: "#C0C0C0";
+			font-weight: "bold";
+		}
+		vwc-side-drawer#side-drawer{
+			height: calc(100vh - 64px);
+			--side-drawer-z-index:3;
+		}
+	</style>
+`;
+
+const loremIpsum = () => html`
+<vwc-layout gutters="md">
+	<div>
```

Comment:
we can wrap the text with `<vwc-text font-face="body-1">` you can add p element inside if you like. Will work without it as well.